src/CLI/CLI.php

```diff
@@ -215,16 +215,16 @@ public static function setResource(string $name, callable $callback, array $inje
     /**
      * task-name --foo=test
      *
-     * @param array $args
+     * @param array<int,string> $args
      * @throws Exception
-     * @return array
+     * @return array<string>
```

Comment:
I don't think this returns `array<string>` considering `$output` is returned and `$output` is used like `$output[$key][] = $value;` and `$output[$key] = array_shift($output[$key]);`